src/antsibull/tagging.py

```diff
@@ -0,0 +1,135 @@
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2022 Maxwell G <gotmax@e.email>
+
+"""
+Validate that collections tag their releases in their respective git repositories
+"""
+import asyncio
+import os
+import re
+import typing as t
+
+import asyncio_pool  # type: ignore[import]
+
+from antsibull_core.logging import log
+
+from antsibull_core import app_context
+from antsibull_core.dependency_files import DepsFile
+from antsibull_core.yaml import store_yaml_file, load_yaml_file
+
+from antsibull.collection_meta import CollectionsMetadata
+
+mlog = log.fields(mod=__name__)
+
+
+def validate_tags_command() -> int:
+    app_ctx = app_context.app_ctx.get()
+    if app_ctx.extra['input']:
+        tag_data = load_yaml_file(app_ctx.extra['input'])
+    else:
+        tag_data = asyncio.run(get_collections_tags())
```

Comment:
I guess `asyncio.run` is only available in 3.7+, but it's used elsewhere in this project.

Reply:
It's somewhat funny that nobody noticed that earlier...

Reply:
It looks like there's already compat code for this (https://github.com/ansible-community/antsibull-core/blob/main/src/antsibull_core/compat.py#L16). Should I use that instead or would you accept a separate PR to remove support for Python 3.6 and 3.7 :)?

Reply:
How about:
1. Keep this PR as-is.
2. Create a third PR which uses the correct compatibility code, so that can be merged first and released as a new bugfix release before we drop support for Python 3.6 and 3.7. I'll work on that one if you're not quicker...
3. 3. Create another PR to remove support for Pyhon 3.6 and 3.7 in this repository.

Reply:
I created #459 to use the compat code.

Reply:
I created #460 for bumping the required Python version.